tests/modules/net_vpc_firewall/test_plan.py

```diff
@@ -12,27 +12,32 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
 def test_vpc_firewall_simple(plan_runner):
   "Test vpc with no extra options."
   _, resources = plan_runner()
   assert len(resources) == 3
-  assert set([r['type'] for r in resources]) == set(
-      ['google_compute_firewall'])
-  assert set([r['values']['name'] for r in resources]) == set(
-      ['vpc-ingress-tag-http', 'vpc-ingress-tag-https', 'vpc-ingress-tag-ssh'])
-  assert set([r['values']['project'] for r in resources]) == set(['project'])
-  assert set([r['values']['network'] for r in resources]) == set(['vpc'])
+  assert set([r['type'] for r in resources]) == set(['google_compute_firewall'])
```

Comment:
`set([comprehension]) == {comprehension}` and `set([a,b,c]) == {a,b,c}`

Reply:
pffffffff :)